app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```

Comment:
Good call to make this package, but why not bringing the listItemAdapter and viewPagerAdapter in there?